### PR TITLE
Make Dart |print| calls show up when running on Mac and iOS.

### PR DESCRIPTION
**Pull request by iansf**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
lgtm

File: sky/engine/bindings/builtin_natives.cc [hunk 1]
```diff
@@ -170,6 +174,8 @@ void Logger_PrintString(Dart_NativeArguments args) {
     // In addition to writing to the stdout, write to the logcat so that the
     // message is discoverable when running on an unrooted device.
     __android_log_print(ANDROID_LOG_INFO, "sky", "%.*s", length, chars);
+#elif __APPLE__
+    syslog(LOG_WARNING, "sky: %.*s", (int)length, chars);
```
Comment (eseidelGoogle) [inline sky/engine/bindings/builtin_natives.cc]: Do you want LOG_INFO?

**Comment by eseidelGoogle** [comment]
lgtm

File: sky/engine/bindings/builtin_natives.cc (hunk 1 above)
Comment (abarth) [inline sky/engine/bindings/builtin_natives.cc]: IMHO, we should remove the "sky:" prefix.  We used to have prefixes on other platforms, but they caused problems for teams using Sky.  On linux, for example, we don't have a prefix anymore.

**Pull request merged**